tests/api/entities/resolver/non_strict_mode.test.js

```diff
@@ -4,11 +4,11 @@ const { randomLabel, generateIsbn13 } = require('tests/api/fixtures/entities')
 
 describe('entities:resolve:non-strict mode', () => {
   it('should ignore and report sanitization errors', async () => {
-    const entry = { edition: {} }
+    const entry = { edition: { isbn: '978000000000' } }
```

Comment:
1. I still think documenting empty `edition: {}` is helpful.

2. Testing ` edition: { isbn: '978000000000' }` should return `invalid isbn`, since the requester does send an invalid isbn, which should throw even though some external id claims are found (?)

Reply:
1. addressed in 8cdfa734f
2. It does return `invalid isbn` in the `res.errors`